main.js

```diff
@@ -26,136 +26,99 @@ categoryBoxWrapper.addEventListener('click', activateCategory);
 startActivityButton.addEventListener('click', submitForm);
 
 //Add functions below
+function show(element) {
+  element.classList.remove('hidden');
+}
+
+function hide(element) {
+  element.classList.add('hidden');
+}
+
 function submitForm() {
   event.preventDefault();
   if (checkInputValidity()) {
     createNewActivity();
-  };
-};
+    hide(activityInputForm);
+    show(timerBoxWrapper);
+  }
+}
 
 function checkInputValidity() {
   //Add validation conditional here for activity boxes to make sure one is clicked
-  if (!accomplishInput.value === true) {
+  if (!accomplishInput.value) {
      alert('Please enter description');
      return false;
-  } else if (!minutesInput.value === true || isNaN(minutesInput.value)) {
+  } else if (!minutesInput.value || isNaN(minutesInput.value)) {
      alert('Please enter valid minutes input');
      return false;
-  } else if (!secondsInput.value === true || isNaN(secondsInput.value) || secondsInput.value > 59) {
+  } else if (!secondsInput.value || isNaN(secondsInput.value) || secondsInput.value > 59) {
      alert(`Please enter valid seconds input`);
```

Comment:
@m1073496 I believe we lost some error message coding here or in the HTML/CSS to show an error message below the description, minutes and seconds? Can we redo?

Reply:
@stevecalla I didn't get a chance to reroute the alert messages to the spots below the description. I think we should fix up the CSS on those alert icons/messages first though.

Reply:
on project board